### PR TITLE
BETA: Register kenny.is-a.dev

### PR DESCRIPTION
**Pull request by KennySB-dev**
Added `kenny.is-a.dev` using the [dashboard](https://manage.is-a.dev).

### CODE REVIEW

**Comment by FireBoiDiscord** [comment]
Do i need to do something else? as i did what the manual instructions said, but the docs link doesn't work.

File: domains/kenny [hunk 1]
```diff
@@ -0,0 +1 @@
+        "email": "kennybugeja@gmail.com"
```
Comment (PhoenixOrigin) [inline domains/kenny:1]: This file is invalid; it doesn't have a json file ending and is also invvalid json so please either fix it / delete the file.

**Review by PhoenixOrigin — CHANGES_REQUESTED** [review]

File: domains/kenny (hunk 1 above)
Comment (FireBoiDiscord) [reply domains/kenny:1]: i deleted it

Comment (PhoenixOrigin) [reply domains/kenny:1]: Doesn't show as deleted on here, can you check again please?

**Review by wdhdev — APPROVED** [review]

**Pull request merged**